.github/actions/e2e/setup-cluster/action.yaml

```diff
@@ -115,7 +116,8 @@ runs:
             roleName: karpenter-irsa-${{ inputs.cluster_name }}
             permissionsBoundaryARN: "arn:aws:iam::${{ inputs.account_id }}:policy/GithubActionsPermissionsBoundary"
             permissionPolicyARNs:
-              - "arn:aws:iam::${{ inputs.account_id }}:policy/KarpenterControllerPolicy-${{ inputs.cluster_name }}""""
+              - "arn:aws:iam::${{ inputs.account_id }}:policy/KarpenterControllerPolicy-${{ inputs.cluster_name }}"
+              - "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy""""
```

Comment:
This wasn't succeeding up to this point?

Reply:
It was succeeding to pull Karpenter logs and container logs on MNG nodes. This is to pull kubelet logs from the Karpenter nodes. I was wanting to pull kubelet logs for the windows failures.